accelerate/astro-starter/src/types/quote.ts

```diff
@@ -0,0 +1,17 @@
+export interface QuoteData {
+  id: number
+  quote: string
+  createdAt: string
+}
```

Comment:
We should maybe use the builtin Prisma type for this, but this also works.
```suggestion
export interface QuoteData {
  id: number
  text: string
  createdAt: string
}
```

Reply:
Used these types for now, renamed quote to text.